src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Swish/SwishPaymentsResource.cs

```diff
@@ -0,0 +1,35 @@
+﻿using SwedbankPay.Sdk.Common;
+using SwedbankPay.Sdk.Extensions;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Swish
+{
+    public class SwishPaymentsResource : ResourceBase, ISwishResource
+    {
+        public SwishPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public async Task<ISwishPaymentResponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)